From a6831eb1d1603284a110f39eeb69d6ab2dc3ee43 Mon Sep 17 00:00:00 2001
From: Leonard Crestez <leonard.crestez@nxp.com>
Date: Wed, 7 Mar 2018 22:37:43 +0200
Subject: [PATCH 1020/5242] MLK-12692 ARM: dts: imx6ull-ddr3-arm2-ecspi:
 enable ecspi1 on arm2 board

commit  76d3b575899b5021eb54890a06bc0f6c1f1d8ccf from
https://source.codeaurora.org/external/imx/linux-imx.git

Add new dtb file since ecspi share pins with csi and esai.

Signed-off-by: Robin Gong <yibin.gong@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/Makefile                         |    1 +
 .../arm/boot/dts/imx6ull-14x14-ddr3-arm2-ecspi.dts |   22 ++++++++++++++++++++
 2 files changed, 23 insertions(+)
 create mode 100644 arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-ecspi.dts

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index 4a08e53..804a0e6 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -600,6 +600,7 @@ dtb-$(CONFIG_SOC_IMX6UL) += \
 dtb-$(CONFIG_SOC_IMX6ULL) += \
 	imx6ull-14x14-ddr3-arm2.dtb \
 	imx6ull-14x14-ddr3-arm2-adc.dtb \
+	imx6ull-14x14-ddr3-arm2-ecspi.dtb \
 	imx6ull-14x14-ddr3-arm2-lcdif.dtb \
 	imx6ull-14x14-ddr3-arm2-gpmi-weim.dtb \
 	imx6ull-14x14-ddr3-arm2-uart2.dtb \
diff --git a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-ecspi.dts b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-ecspi.dts
new file mode 100644
index 0000000..f86931f
--- /dev/null
+++ b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-ecspi.dts
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2016 Freescale Semiconductor, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+
+#include "imx6ull-14x14-ddr3-arm2.dts"
+
+&ecspi1 {
+	status = "okay";
+};
+
+&esai {
+	status = "disabled";
+};
+
+&ov5640{
+	status = "disabled";
+};
+
-- 
1.7.9.5

